### ISSUE TITLE
ERROR: Unable to parse a valid provider from guid:plex://

### ISSUE THREAD

**Issue by tyjtyj** [Taxel/PlexTraktSync#363]
### The problem

_A description of what the bug is, including a complete traceback (if applicable)_
run  cd /PlexTraktSync &&  ./plex_trakt_sync.sh sync --sync movies

```
Processing Movies  [################################----]  585
ERROR: Unable to parse a valid provider from guid:plex://movie/5d77686b374a5b001fec51ae

Traceback (most recent call last):
  File "./main.py", line 6, in <module>
    cli()
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.7/dist-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/root/PlexTraktSync/plex_trakt_sync/commands/sync.py", line 161, in sync
    sync_all(walker=w, trakt=trakt, plex=plex)
  File "/root/PlexTraktSync/plex_trakt_sync/commands/sync.py", line 77, in sync_all
    for movie in walker.find_movies():
  File "/root/PlexTraktSync/plex_trakt_sync/walker.py", line 71, in find_movies
    movie = self.mf.resolve_any(plex)
  File "/root/PlexTraktSync/plex_trakt_sync/media.py", line 105, in resolve_any
    for guid in pm.guids:
  File "/root/PlexTraktSync/plex_trakt_sync/plex_api.py", line 116, in guids
    ordered = sorted(guids, key=lambda guid: sort_order[guid.provider])
  File "/root/PlexTraktSync/plex_trakt_sync/plex_api.py", line 116, in <lambda>
    ordered = sorted(guids, key=lambda guid: sort_order[guid.provider])
KeyError: 'plex'
```

### Expected behavior

_A description of what you expected to happen_
No error. Unable to determine which movie cause that

### Steps to reproduce the behavior

_With a complete code example, if possible_
run  cd /PlexTraktSync &&  ./plex_trakt_sync.sh sync --sync movies
### Workarounds

_Is there an existing workaround for this issue?_
No
### Environment

- Version: 0.12.18
- Python version: 3.7.3
- Platform: Rasbian 10

**Comment by tyjtyj** [comment]
Check my file seems already patch with https://github.com/Taxel/PlexTraktSync/pull/359/files

but still throwing error.

**Comment by tyjtyj** [comment]
seems fix after force update to PlexTraktSync [c9dd0eee: Cleanup unused imports @2021-06-24 01:50:10]
